### DISCUSSION TITLE
Array of rpc urls

### DISCUSSION THREAD

**Discussion by LeoMatiusso** [WalletConnect/walletconnect-monorepo#2881] [Developer Support] [answered]
Using the Ethereum Provider, is there a way to pass an array, not a map, of rpc urls? I couldn't find anything in previous questions or in the documentation.

**Comment by ignaciosantise** [comment, marked as answer]
Hey @LeoMatiusso 👋 Our ethereum provider only accepts a map of urls. 

You can check the interface here: https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/providers/ethereum-provider/src/EthereumProvider.ts#L52

example:
```
rpcMap: {
  1: "https://eth.llamarpc.com",
  80001: "https://rpc.walletconnect.com?chainId=eip155:80001&projectId=<your walletconnect project id>",
},
```
